content/changelog/2022-10-04-console-release/index.md

```diff
@@ -0,0 +1,15 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Social authentication with Hasura provider.
+- Control Plane: Improved performance of project creation.
+
+### Bug fixes
+
+- UI: Database selector allows to select any of created databases.
```

Comment:
Is this the database drop-down on the Dashboard tab, under Connection Details?

Reply:
@seymourisdead where [this drop-down](https://github.com/neondatabase/cloud/pull/2417/commits/5a9277a2d1e0a29ef179593200b6bd5ab7f587fd) located?

Reply:
in the Connection details and on the SQL Editor page